com.unity.render-pipelines.high-definition/Documentation~/whats-new-12.md

```diff
@@ -15,6 +15,10 @@ Group of Materials / GameObject can be setup to use Force Emissive forward with
 
 ## Improvements
 
+### Area Lights
+
+Area lights are now correctly supported for Fabric and Hair master node as well as for AxF shader.
```

Comment:
The AxF shader and Fabric and Hair master nodes now correctly support Area lights.